### PR TITLE
Single user csv import error

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Fix importing user/users columns when exported as CSV. The current logic is handling CSV strings as JSON, but this is not true for our CSV exported objects. Adding some tests to check that the code works as expected.

## Launchcontrol
Fix importing user/users data

## Feature branch env
[Feature Branch Link](http://fb-budi-8396-single-user-csv-import-error.fb.qa.budibase.net)

### CODE REVIEW

File: packages/server/src/api/controllers/view/exporters.ts [hunk 1]
```diff
@@ -17,21 +17,22 @@ export function csv(
...
 
+export function valueToCsv(val: any) {
+  val =
+    typeof val === "object" && !(val instanceof Date)
+      ? `"${JSON.stringify(val).replace(/"/g, "'")}"`
+      : val !== undefined
+      ? `"${val}"`
+      : ""
+  return val.trim()
+}
```
[the remark this answers is not in the document]
Comment (adrinr) [reply packages/server/src/api/controllers/view/exporters.ts:34]: Yes... there is, but the current exports are generated via this function and that could add a breaking change...

[the remark this answers is not in the document]
Comment (samwho) [reply packages/server/src/api/controllers/view/exporters.ts:34]: 😭

File: packages/server/src/api/controllers/view/tests/exporters.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { generator } from "@budibase/backend-core/tests"
+import { valueToCsv, parseCsvExport } from "../exporters"
+
+describe("exporters", () => {
+  describe("export and import", () => {
+    it.each([
+      ["string", generator.word()],
```
[the remark this answers is not in the document]
Comment (adrinr) [reply packages/server/src/api/controllers/view/tests/exporters.spec.ts]: Not needed anymore

File: packages/server/src/api/controllers/view/exporters.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrinr) [reply packages/server/src/api/controllers/view/exporters.ts]: Refactored. As discussed, our current implementation is not working fine, so adding breaking changes is not a problem as it's currently not working for these use cases

**Comment by adrinr** [comment]
Closed in favour of https://github.com/Budibase/budibase/pull/14302

**Pull request closed without merging**